packages/react-core/src/components/DataList/DataListAction.tsx

```diff
@@ -35,12 +33,10 @@ export const DataListAction: React.FunctionComponent<DataListActionProps> = ({
   id,
   'aria-label': ariaLabel,
   'aria-labelledby': ariaLabelledBy,
-
-  isPlainButtonAction,
   ...props
 }: DataListActionProps) => (
   <div className={css(styles.dataListItemAction, formatBreakpointMods(visibility, styles), className)} {...props}>
-    {isPlainButtonAction ? <div className={css(styles.dataListAction)}>{children}</div> : children}
+    <div className={css(styles.dataListAction)}>{children}</div>
```

Comment:
This should be good to update now that https://github.com/patternfly/patternfly/pull/7089 has been merged in core